### PR TITLE
docs: update the multi-disk management

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
- For newly added provisioner and reference

### CODE REVIEW

File: docs/host/host.md [hunk 1]
```diff
@@ -184,26 +184,41 @@ Users can view and add multiple disks as additional data volumes from the edit h
...
 
 :::caution
 
 As of Harvester v1.0.2, we no longer support adding partitions as additional disks. If you want to add it as an additional disk, be sure to delete all partitions first (e.g., using `fdisk`).
 
 :::
 
-4. Select an additional raw block device to add as an additional data volume.
-    - The `Force Formatted` option is required if the block device has never been force-formatted.
+4. In v1.4.0, we introduce the provisioner concept to manage the additional disks. You can choose the provisioner for the disk.
```
Comment (jillian-maroket) [inline docs/host/host.md:199]:
```suggestion
4. Select a provisioner for the disk.
```

File: docs/host/host.md [hunk 2]
```diff
@@ -184,26 +184,41 @@ Users can view and add multiple disks as additional data volumes from the edit h
...
 
-![Force Format](/img/v1.2/host/force-format-disks.png)
+- The `LonghornV1` provisioner is the default provisioner like before.
```
Comment (jillian-maroket) [inline docs/host/host.md:201]:
```suggestion
  - **LonghornV1 (CSI)**: This is the default provisioner.
```

File: docs/host/host.md [hunk 3]
```diff
@@ -184,26 +184,41 @@ Users can view and add multiple disks as additional data volumes from the edit h
...
 
-5. Last, you can click **⋮ > Edit Config** again to check the newly added disk. Meanwhile, you can also add the "Host/Disk" tag (details are described in the [next section](#storage-tags)).
+![Provisioner LonghornV1](/img/v1.4/host/multidisk-mgmt-03.png)
```
Comment (jillian-maroket) [inline docs/host/host.md:203]:
```suggestion
    ![Provisioner LonghornV1](/img/v1.4/host/multidisk-mgmt-03.png)
```

File: docs/host/host.md [hunk 4]
```diff
@@ -184,26 +184,41 @@ Users can view and add multiple disks as additional data volumes from the edit h
...
 
-![Check Result](/img/v1.2/host/check-added-disks.png)
+  - The `Force Formatted` option is required if the block device has never been force-formatted.
+
+  ![Force Format](/img/v1.4/host/multidisk-mgmt-08.png)
```
Comment (jillian-maroket) [inline docs/host/host.md:207]:
```suggestion
    You must set **Force Formatted** to **Yes** if the block device has never been force-formatted.

    ![Force Format](/img/v1.4/host/multidisk-mgmt-08.png)
```

File: docs/host/host.md [hunk 5]
```diff
@@ -184,26 +184,41 @@ Users can view and add multiple disks as additional data volumes from the edit h
...
+
+- The `LonghornV2` provisioner is a new provisioner that support the new Longhorn v2 engine. You can find more details in the [Longhorn V2 Data Engine](../advanced/longhorn-v2.md).
+
+![Provisioner LonghornV2](/img/v1.4/host/multidisk-mgmt-04.png)
```
Comment (jillian-maroket) [inline docs/host/host.md:211]:
```suggestion
  - **LonghornV2 (CSI)**: Select this provisioner if you want to use the [Longhorn V2 Data Engine](../advanced/longhorn-v2.md).

    ![Provisioner LonghornV2](/img/v1.4/host/multidisk-mgmt-04.png)
```

File: docs/host/host.md [hunk 6]
```diff
@@ -184,26 +184,41 @@ Users can view and add multiple disks as additional data volumes from the edit h
...
+
+- The `LVM` provisioner is a new provisioner that support the LVM local storage. You can find more details in the [LVM Local Storage](../advanced/addons/lvm-local-storage.md).
+
+![Provisioner LVM](/img/v1.4/host/multidisk-mgmt-05.png)
```
Comment (jillian-maroket) [inline docs/host/host.md:215]:
```suggestion
  - **LVM**: Select this provisioner if you want to use [local storage](../advanced/addons/lvm-local-storage.md) to create persistent volumes for your workloads.

    ![Provisioner LVM](/img/v1.4/host/multidisk-mgmt-05.png)
```

File: docs/host/host.md [hunk 7]
```diff
@@ -184,26 +184,41 @@ Users can view and add multiple disks as additional data volumes from the edit h
...
+
+5. Last, you can click **⋮ > Edit Config** again to check the newly added disk. Meanwhile, you can also add the "Host/Disk" tag (details are described in the [next section](#storage-tags)). It only supports the `LonghornV1` and `LonghornV2` provisioner.
+
+![disk tag 01](/img/v1.4/host/multidisk-mgmt-06.png)
+
+![disk tag 02](/img/v1.4/host/multidisk-mgmt-07.png)
```
Comment (jillian-maroket) [inline docs/host/host.md:221]:
```suggestion
5. Click **Save**.

6. On the host details screen, verify that the disks were added and the correct provisioner was set.

You can also add [storage tags](#storage-tags) if you want Longhorn volume data to be stored on specific nodes or disks. Storage tags can only be used with the **LonghornV1 (CSI)** and **LonghornV2 (CSI)** provisioners.

![disk tag 01](/img/v1.4/host/multidisk-mgmt-06.png)

![disk tag 02](/img/v1.4/host/multidisk-mgmt-07.png)
```

**Review by bk201 — APPROVED** [review]
LGTM. Thanks for adding such a good introduction section.
One nit is screenshots are captured in dark mode and they are not consistent with other screenshot in the same page.

File: docs/host/host.md (hunk 2 above)
Comment (jillian-maroket) [reply docs/host/host.md:201]: Ideally, we should explain why users should select this provisioner. We do that for LonghornV2 and LVM.

**Review by jillian-maroket — COMMENTED** [review]
Review done

**Review by jillian-maroket — APPROVED** [review]

**Review by tserong — APPROVED** [review]

**Pull request merged**